Resources/Prototypes/_Mono/Shipyard/BlackMarket/mercury.yml

```
@@ -0,0 +1,32 @@
+- type: vessel
+  id: Mercury
+  parent: BaseVesselAntag
+  name: Mercury
+  description: A pirate ship fit for up-close brawls. Equipped with 4x ASM-302 and 4x AK-570.
+  price: 78520
+  category: Medium
+  group: BlackMarket
+  shuttlePath: /Maps/_Mono/Shuttles/BlackMarket/mercury.yml
+  guidebookPage: Null
+  class:
+  - Pirate
```

YAMLlint (1.35.1)

[error] wrong indentation: expected at least 3

(indentation)